project/plugins.sbt

```diff
@@ -0,0 +1,6 @@
+resolvers ++= Seq(
+  Resolver.sonatypeRepo("releases"),
+  Resolver.bintrayRepo("beyondthelines", "maven")
```

Comment:
Do we need the `beyondthelines` resolver?

Reply:
Let me do some adjustments to avoid it, 👍